templates/katello.yaml.erb

```diff
@@ -31,6 +31,10 @@
     :url: <%= @qpid_url %>
     :subscriptions_queue_address: <%= @candlepin_event_queue %>
 
+  :container_image_registry:
```

Comment:
I'm thinking about container image registry vs container registry but I'm not sure which would be better.

Reply:
I prefer as is but either is fine with me. We call the resource "container image" throughout the UI now (as we've tried to downplay the term _docker_).